integration_util_test.go

```diff
@@ -0,0 +1,40 @@
+package deck
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/k1LoW/errors"
+	"google.golang.org/api/slides/v1"
+)
+
+// ListSlideURLs lists URLs of the slides in the Google Slides presentation.
+func (d *Deck) ListSlideURLs() []string {
```

Comment:
These functions are certainly not used as a CLI, but they are exported in the belief that they may be used as pkg.
(Also, as you can imagine, they are for testing purposes 💦 )

I think it would be better to take it out of `*_test.go` to eliminate confusion.